plugin/plugin.go

```diff
@@ -129,6 +133,11 @@ func (p *OrmPlugin) Generate(file *generator.FileDescriptor) {
 				typeName := generator.CamelCase(msg.GetName())
 				if p.isOrmable(typeName) {
 					p.parseAssociations(msg)
+					o := p.getOrmable(typeName)
+					if p.hasPrimaryKey(o) {
+						_, fd := p.findPrimaryKey(o)
+						fd.BindTo = o.OriginName
```

Comment:
I guess we don't need this `Parent` information in a field if it's not a foreign key. 
In Converters you may check if `Parent` is not nil, then use `Parent`s type name, otherwise use current type name.

Reply:
That is needed for primary as well to decode/encode self id based on right proto type. If I remove it I would need to add complexity during generation.
Now I know if `fd.BindTo == ""` it is an external identifier.